corelib/EAST/EAST_utils.py

```diff
@@ -0,0 +1,179 @@
+import numpy as np
+import cv2
+import lanms
+
+from logger.logging import RekogntionLogger
+
+
+logger = RekogntionLogger(name="EAST_utils")
+
+
+def preprocess(im, max_side_len=2400):
+    """     Resize image to make it a multiple of 32
+    Args:
+            *   im: input image
+            *   max_side_len: maximum allowed side length
+    Workflow:
+            *   Ratio is set to 1 if none of the dimension exceeds
+                the maximum allowed side length, else it is set to
+                quotient obtained on dividing max_side_len with the
+                dimesion that is greater.
+            *   Both dimensions are multiplied by the ratio, scaling
+                the image down by that factor
+            *   image is then resized such that both dimensions are
+                equal to the closest multiple of 32
+            *   resizing factor is calculated
+    Returns:
+            *   Rezised Image along  with resizing factor
+    """
+
+    logger.info(msg="preprocess called")
+    h, w, _ = im.shape
+
+    resize_w = w
+    resize_h = h
+
+    if max(resize_h, resize_w) > max_side_len:
+        ratio = float(max_side_len) / resize_h if resize_h > resize_w else float(max_side_len) / resize_w
+    else:
+        ratio = 1.
+    resize_h = int(resize_h * ratio)
+    resize_w = int(resize_w * ratio)
+
+    resize_h = resize_h if resize_h % 32 == 0 else (resize_h // 32) * 32
+    resize_w = resize_w if resize_w % 32 == 0 else (resize_w // 32) * 32
+    im = cv2.resize(im, (int(resize_w), int(resize_h)))
+
+    ratio_h = resize_h / float(h)
+    ratio_w = resize_w / float(w)
+
+    return im, (ratio_h, ratio_w)
+
+
+def restore_rectangle(origin, geometry):
+
+    logger.info(msg="restore_rectangle called")
+    d = geometry[:, :4]
```

Comment:
give better variable names similarly to wherever it is required.

Reply:
have a look